src/components/MonitorList.vue

```diff
@@ -10,7 +10,7 @@
                     <font-awesome-icon icon="times" />
                 </a>
                 <form>
-                    <input v-model="searchText" class="form-control search-input" :placeholder="$t('Search...')" autocomplete="off" />
+                    <input v-model="searchText" class="form-control search-input" :placeholder="$t('Search...')" autocomplete="off" aria-label="Search monitored sites" />
```

Comment:
What do you think about such a change (plus adding this to the translation file(s) you can)

```suggestion
                    <input v-model="searchText" class="form-control search-input" :placeholder="$t('Search...')" autocomplete="off" :aria-label="$t('Search monitored sites')" />
```

Reply:
On my side, i agree with this suggestion.